haskell-template.cabal

```diff
@@ -87,7 +85,6 @@ common shared
     , profunctors
     , relude
```

Comment:
This should be `relude >= 1.0`, right? Because lower versions won't export these modules.

Reply:
updated